app/src/models/pull-request.ts

```diff
@@ -25,6 +25,17 @@ export class PullRequestRef {
   }
 }
 
+/** The commit status and metadata for a given ref */
+export class CombinedRefStatus {
```

Comment:
We don't seem to ever instantiate this class. I'm betting this code happens to work currently because of TS' structural type checking, but we don't seem to be actually using this.

Reply:
Yea, that is what's happening, I added a constructor so that should solve the structural type checking TS does.

Reply:
Let's push the question further: is there a reason we need this class? It seems like we're just copying values to/from it.